src/views/UIOptionsPopup/UIOptionsPopup.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { useGlobalState } from '../../modules/global'
+import { UIOptionsDialog } from './style'
+import ValueSlider from '../Controls/ValueSlider'
+import ValueCheckbox from '../Controls/ValueCheckbox'
+import { Row, Col } from 'antd'
+import { CloseSquareOutlined } from '@ant-design/icons'
+
+type Props = {
+  children?: React.ReactNode
+}
+
+const UIOptionsPopup: React.FC<Props> = ({ children }: Props) => {
+  const [isVisible, setVisibility] = useGlobalState('isUIOptionsPopupVisible')
+
+  const showPopup = () => setVisibility(true)
+  const hidePopup = () => setVisibility(false)
+
+  const optionsDialog = isVisible ? (
+    <UIOptionsDialog>
+      <CloseSquareOutlined
+        style={{ float: 'right', zoom: 1.75, position: 'relative', right: '-1px' }}
+        onClick={hidePopup}
+      />
+      <ValueSlider
+        stateKey="orbitSpeed"
+        min={0.1}
+        max={1}
+        step={0.1}
+        sliderColumns={18}
+        spinnerColumns={4}
+        label={'Rotation Sensitivity'}></ValueSlider>
+      <div style={{ margin: '2px 8px', height: '0.9em', top: '-0.6em', position: 'relative' }}>
+        <ValueCheckbox stateKey={'showGrid'} label="Show Grid" />
+      </div>
+    </UIOptionsDialog>
+  ) : (
+    <></>
+  )
+
+  return (
+    <div onContextMenu={showPopup} style={{ height: 'inherit', width: 'inherit}' }}>
```

Comment:
Interesting thought putting it on a context menu trigger, however I don't think that's necessary for this. First off, people wouldn't intuitively know to do that (I had to look at the code to figure it out) and second people often get cranky when they have their regular context menus overridden in the browser. To that end, I would just use a settings icon from [antd](https://ant.design/components/icon/) and make a button out of it that triggers the menu. I'd put that button either in the toolbar by the menu button, in the upper right of the settings panel, or maybe just make it another button with the social media. Play around and see what looks best.

Reply:
I added a configuration button, and removed the context menu trigger.